content/tutorials/manage-the-rippled-server/configuration/configure-advisory-deletion.md

```diff
@@ -0,0 +1,96 @@
+
+- A `cron` daemon is installed and running.
+
+    Ubuntu Linux runs a `cron` daemon by default.
+
+    On RHEL or CentOS, you can install the `cronie` package:
+
+        $ sudo yum install cronie
+
+- Your server has enough disk space to store your chosen amount of history in its ledger store.
+
+    See [Capacity Planning](capacity-planning.html) for details of how much storage is required for different configuration. With advisory deletion enabled, the maximum history a server may accumulate before deletion is equal to the number of ledger versions configured in the `online_delete` setting **plus** the amount of time between online deletion prompts.
+
+- You know which hours are least busy for your server.
+
+## Configuration Steps
+
+To configure advisory deletion with a daily schedule, perform the following steps:
+
+1. Enable `advisory_delete` in the `[node_db]` stanza of your `rippled`'s config file.
+
+        [node_db]
+        # Other settings unchanged ...
+      	online_delete=2000
+      	advisory_delete=1
+
+    - Set `advisory_delete` to `1` to require approval before running online deletion. (Set it to `0` to run online deletion automatically as new ledger versions become available.)
+    - Set `online_delete` to the minimum number of ledger versions to keep after running online deletion. The server accumulates more history than this until online deletion runs.
+
+    {% include '_snippets/conf-file-location.md' %}<!--_ -->
+
+2. Test running the [can_delete method][] to prompt the server to run online deletion.
+
+    You can use the [`rippled` commandline interface](get-started-with-the-rippled-api.html#commandline) to run this command. For example:
+
+        $ rippled --conf=/etc/opt/ripple/rippled.cfg can_delete now
+
+    The response indicates the maximum ledger index that may the server may delete from its ledger store. For example, the following message indicates that ledger versions up to and including ledger index 43633667 are to be deleted:
+
+        {
+          "result": {
+            "can_delete": 43633667,
+            "status": "success"
+          }
+        }
+
+3. Configure your `cron` daemon to run the `can_delete` method you tested in the previous step at a scheduled time.
+
+    Edit your `cron` configuration:
+
+        $ crontab -e
+
+    The following example sets the server to run deletion at 1:05 AM server time daily:
+
+        5 1 * * * rippled --conf /etc/opt/ripple/rippled.cfg can_delete now
+
+    Be sure that you schedule the command to run based on the time zone your server's clock is configured to use. Ripple recommends running all production servers with the UTC time zone.
+
+    **Tip:** You do not need to schedule a `cron` job to run online deletion if you have `advisory_delete` disabled. In that case, `rippled` runs online deletion automatically when the server's oldest available ledger is twice as old as the number of ledgers to keep after deletion.
```

Comment:
Actually, it's triggered once the difference between the earliest ledger acquired and the latest level acquired are at least the value of online_delete. This also means that, eventually, the oldest available ledger will be roughly twice that value.